testsuite/integration-tests/pom.xml

```diff
@@ -61,48 +65,43 @@
                 <filtering>true</filtering>
             </resource>
         </resources>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration combine.children="append">
+                    <excludedGroups>${intersmash.test.excludedGroups.by.execution-profile},${intersmash.test.excludedGroups.by.wildfly-target-distribution}</excludedGroups>
+                </configuration>
+            </plugin>
+        </plugins>
     </build>
 
     <profiles>
         <profile>
-            <id>ts.prod</id>
+            <id>ts.execution-profile.prod</id>
             <properties>
                 <!-- Set the intersmash.test.execution.profile property to `prod`, it is read by test classes -->
                 <intersmash.test.execution.profile>prod</intersmash.test.execution.profile>
                 <!-- Exclude tests that shouldn't be executed when dealing with products -->
-                <intersmash.test.execution.excludedGroups>ts.not-for-prod</intersmash.test.execution.excludedGroups>
+                <intersmash.test.excludedGroups.by.execution-profile>ts.not-for-prod</intersmash.test.excludedGroups.by.execution-profile>
             </properties>
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-surefire-plugin</artifactId>
-                        <configuration combine.children="append">
-                            <excludedGroups>${intersmash.test.execution.excludedGroups}</excludedGroups>
-                        </configuration>
-                    </plugin>
-                </plugins>
-            </build>
         </profile>
         <profile>
-            <id>ts.community</id>
+            <id>ts.execution-profile.community</id>
```

Comment:
What is the reason for having this profile? My understanding is that this provide just overrides default values with the same values. Can we remove this profile? Or remove definitions of intersmash.test.execution.profile and intersmash.test.excludedGroups.by.execution-profile from defaults and enabling this profile automatically once ts.wildfly.target-distribution.eap won't be enabled?

Reply:
+1, removing it and leaving defaults.